modules/plugins/repl/conjure/conjure.nix

```
@@ -0,0 +1,19 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}: let
+  inherit (lib.options) mkEnableOption;
+  inherit (lib.modules) mkIf;
+
+  cfg = config.vim.repl.conjure;
+in {
+  options.vim.repl.conjure = {
+    enable = mkEnableOption "Whether to enable Conjure";
```

```suggestion
    enable = mkEnableOption "Conjure";
```

mkEnableOption already prepends some text like "enable ..."